example/pyproject.toml

```
@@ -9,6 +9,7 @@ readme = "README.md"
 python = "^3.11"
 realtime = {path = "..", develop = true}
 python-dotenv = "^1.0.1"
+requests = "*"
```

```suggestion
requests = "^2.32.0"
```

More "future proof" ? 🤔